tests/Resources/ti.ui.scrollableview.addontest.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-2018 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.ScrollableView', function () {
+	this.timeout(5000);
+
+	const isiOS14 =  (parseInt(Ti.Platform.version.split('.')[0]) >= 14);
+	let win;
+	afterEach(done => { // fires after every test in sub-suites too...
+		if (win && !win.closed) {
+			win.addEventListener('close', function listener () {
+				win.removeEventListener('close', listener);
+				win = null;
+				done();
+			});
+			win.close();
+		} else {
+			win = null;
+			done();
+		}
+	});
+
+	it.ios('preferredIndicatorImage', function (finish) {
+		if (!isiOS14) {
+			finish();
```

Comment:
```suggestion
			return finish();
```
Otherwise it will continue on